Project-3/Jvonderh Project 3

```diff
@@ -0,0 +1,228 @@
+  (c)  Asymmetric
+ Has role in modeling
+  This should not be asymmetric because, while a biological, experimental, or computational artifact can be used to model a separate entity, we may also use the entity to model the artifacts. We may be interested in modeling entities through the artifacts, but we may also be interested in using the entities to understand the artifacts. In that way, for example, we may use the actions of real life agents to understand the decisions and considerations used to produce and utilize a computational agents. 
+  (d)  Functional 
+  Characteristic of
+  This should not be functional because, in a relationship between a characteristic and its bearer where the characteristic depends on the bearer for its existence, it is possible for a characteristic to have multiple bearers. Since a functional object property requires that it only output only one possible bearer in this case, it would contradict the property for it to have multiple bearers. As an example, the characteristic "weighs 50 pounds" will have multiple bearers. This may include boxes of office supplies, a piece of furniture, etc.,  but it will mean that this characteristic is a characteristic of multiple bearers. This means that there is more than one value for the characteristic of relation, meaning that it should not be functional.
+  (e)  Inverse Functional
+  Has characteristic
+  This should not be inverse functional because, in a relationship where a bearer has a characteristic which depends on the bearer for its existence, there will be more than one characteristic which the bearer will have. Since an inverse functional object property requires that there be only one input in the form of a characteristic into the bearer, for any characteristic at most one bearer can bear it. As an example, it is possible for multiple people to bear the characteristic of "has a beard." This will mean that the has characteristic relation will have more than a value of 1 or 0. As such, the has characteristic relation should not be inverse functional. 
+```
+
+3. Model the following natural language expressions using terms from BFO and RO; you are welcome to introduce new terms where needed:  
+```
+  (a) Sally has an arm Tuesday but does not have an arm Wednesday.
+  Sally is instance_of object
+Tuesday is instance_of one-dimensional temporal region 
+Wednesday is an instance of one-dimensional temporal region 
+Arm is an instance of fiat object part
+“Sally participates in having at least one arm on Tuesday” is an instance of occurrent 
```

Comment:
Not quite, your model should look a little more like: 
Sally is an instance of object
Sally participates in a process p of having an arm
p has a temporal extent that is Tuesday...